### DISCUSSION TITLE
Please fix repository indicator stuck at 5K

### DISCUSSION THREAD

**Discussion by standardgalactic** [community/community#128722] [Repositories] [answered]
### Select Topic Area

Bug

### Body

I am finding it annoying that my number of repositories is stuck at 5K+ I have like 18 thousand repositories.  It looks like Microsoft's page says 6.2K so I know it's possible.  I haven't seen any other accounts with more than 5 thousand repositories, so maybe this problem isn't affecting anyone else. Could someone please fix this?

**Comment by arudeep15** [comment, marked as answer]
If you’re encountering a repository limit stuck at 5,000 repositories, it might be due to account limitations or restrictions on your platform

1. Check Account Type and Limits
2. Upgrade Your Plan
3. Archive or Delete Unnecessary Repositories
4. Contact GitHub Support

**Reply by standardgalactic** [reply]
Thank you. I haven't hit an actual limit, it's just the indicator on my profile. I am on the free tier. I can easily check the number of repositories with:
gh api users/standardgalactic | jq '.total_private_repos + .public_repos'

**Reply by arudeep15** [reply]
could you please mark my answer would be highly appreciated
